tarpaulin.toml

```diff
@@ -22,6 +22,4 @@ exclude = [
     # repo automation (ci, codegen)
     "uniffi-bindgen",
     "xtask",
-    # temporary: tiny test suite, expensive to run
-    "sliding-sync-integration-test",
```

Comment:
Since sliding sync tests are no longer excluded for coverage collecting, we need to set up the proxy in `coverage.yml`, the same way it's set up in `ci.yml` / `sliding-sync-integration-tests` (which also needs merging with `integration-tests`).